tests/unit/mixins/ember-cp-validations-helper-test.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import EmberCpValidationsHelperMixin from 'ember-rapid-forms/mixins/ember-cp-validations-helper';
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('mixin:ember-cp-validations-helper', 'Unit | Mixin | ember cp validations helper');
+
+// Replace this with your real tests.
```

Comment:
Yes definitely more tests are needed. But I've got no good idea what they should test. Maybe an object with some fake validations on it an whether they will correctly render. But this should be integration tests, no?